generate-adaptors/index.js

```diff
@@ -193,25 +215,51 @@ module.exports = function (context, { apiUrl }) {
       cli
         .command('generate-adaptors')
         .description('Generate documentation for OpenFn adaptors')
-        .action(async () => {
+        .option(
+          '-m',
+          '--monorepo <path>',
+          'use the adaptors monorepo to load docs metadata'
+        )
+        .action(async cmd => {
           fs.existsSync('./adaptors/packages') ||
             fs.mkdirSync('./adaptors/packages');
 
-          console.log('Getting version list...');
-          await listVersions();
+          let useMonorepo = false;
+
+          // the monorepo option is suppposed to be written to cmd
+          // ... but it's not.
+          // This is a total hack but it works
+          const last = process.argv.at(-1);
+          if (last === '-m' || last === '--monorepo') {
+            useMonorepo = true;
+          }
 
-          console.log(`Found ${versions.length} monorepo versions.`);
+          if (!useMonorepo) {
+            console.log('Getting version list...');
+            await listVersions();
 
-          fs.writeFileSync(
-            './adaptors/packages/versions.json',
-            JSON.stringify(versions, null, 2)
-          );
+            console.log(`Found ${versions.length} monorepo versions.`);
 
-          const adaptors = await loadAdaptorsDocs();
+            fs.writeFileSync(
+              './adaptors/packages/versions.json',
+              JSON.stringify(versions, null, 2)
+            );
+          } else {
+            console.warn('Skipping version list as loading from monorepo');
+          }
+
+          const adaptors = await (useMonorepo
+            ? loadAdaptorsDocsFromMonorepo(process.env.OPENFN_ADAPTORS_REPO)
+            : loadAdaptorsDocsFromGithub());
 
           console.log('Generating adaptors docs via JSDoc...');
 
           adaptors.map(a => {
+            if (!a.name) {
```

Comment:
Oh this is unrelated - I started getting weird errors locally for `cartodb`. I figure it's worth leaving this here just for better devx later